tools/ppchcalls.py

```diff
@@ -117,12 +118,18 @@
     644: 'H_ADD_CONN',
     648: 'H_DEL_CONN',
     664: 'H_JOIN',
+    672: 'H_VASI_SIGNAL',
     676: 'H_VASI_STATE',
+    680: 'H_VIOCTL',
     688: 'H_ENABLE_CRQ',
     696: 'H_GET_EM_PARMS',
     720: 'H_SET_MPP',
     724: 'H_GET_MPP',
+    732: 'H_REG_SUB_CRQ',
     748: 'H_HOME_NODE_ASSOCIATIVITY',
+    736: 'H_FREE_SUB_CRQ',
+    740: 'H_SEND_SUB_CRQ',
+    744: 'H_SEND_SUB_CRQ_INDIRECT',
```

Comment:
I guess you wanted to place these before 748 above ?

Reply:
@planetharsh that's right, i missed keeping this in sorted order. fixed it now. Thanks.